cmake/setup_finalize.cmake

```diff
@@ -80,6 +80,30 @@ FOREACH(_suffix ${DEAL_II_LIST_SUFFIXES})
 ENDFOREACH()
 
 #
+# Sanity check: Can we compile with the final setup?
+#
+
+FOREACH(build ${DEAL_II_BUILD_TYPES})
+  CHECK_COMPILER_SETUP(
+    "${DEAL_II_CXX_FLAGS} ${DEAL_II_CXX_FLAGS_${build}}"
+    "${DEAL_II_CXX_FLAGS} ${DEAL_II_CXX_FLAGS_${build}}"
```

Comment:
I just checked it on my system and it took me a while to realize to find the configuration that should fail. It should read DEAL_II_LINKER_FLAGS in the two places of the second line, shouldn't it?